ehr/src/org/labkey/ehr/EHRController.java

```diff
@@ -375,8 +376,32 @@ public ModelAndView getView(EHRQueryForm form, BindException errors)
                         url.addParameter("importURL", importUrl.toString());
                     }
                 }
-                else
+                //TODO implement this after view is set up for editable react form
+                else if (isReactForm(form.getSchemaName(), form.getQueryName()))
                 {
+                    //weight is custom since it was introduced in its own controller
+                    if ("weight".equals(form.getQueryName())){
```

Comment:
We should find an approach that doesn't require hard-coding of WNPRC in general, or of the oddities of the weights form in particular. Could isReactForm() return the URL to use based on the values in the TABLE_FORM_FRAMEWORK_TYPES table?